views/popups.pug

```diff
@@ -62,19 +62,19 @@
           | preserved as part of shared URLs, and are persisted locally using browser
           | local storage.
           h4 Editor
-          .form-group(role="group")
-            .form-control.checkbox
-              label
-                input.autoCloseBrackets(type="checkbox")
-                | Automatically insert matching brackets and parentheses
-            .form-control Delay before compiling:&nbsp;
-              b Disabled
-              .slider.slider-horizontal.delay
-              b 3s
-            .form-control.checkbox
-              label
-                input.hoverShowSource(type="checkbox")
-                | Highlight linked code lines on hover
+            .form-group(role="group")
```

Comment:
Good spot! Really it should be undented not indented (the "compilation" h4 is the one that's wrong here! whoops. Will fix on master!)